docs/connect.html

```diff
@@ -252,6 +252,12 @@ <h4><a id="connect_rest" href="#connect_rest">REST API</a></h4>
         <li><code>PUT /connector-plugins/{connector-type}/config/validate</code> - validate the provided configuration values against the configuration definition. This API performs per config validation, returns suggested values and error messages during validation.</li>
     </ul>
 
+    <p>Finally, supported REST requests at the top-level endpoint are:</p>
```

Comment:
suggest: `The following is a supported REST request at the top-level endpoint:`

Reply:
@soenkeliebau , yes. I noticed there's a semantic (yet maybe incidental) grouping of the endpoints in groups of `/connector/` endpoints, `/connector-plugins/` and now `/`. 

To me it seems that, for the time being, this reads better than attaching the missing doc with any of the other groups. 

Thanks @tobydrake7. Incorporated.

Reply:
When reading the previous sections I wondered about that grouping as well. It is not mentioned in the text anywhere and seems to be mostly coincidental. 
I wonder whether it might make sense to just pull all of them together?

Reply:
I agree. But all the endpoints are still under the same paragraph no far apart with each other, and the text reads well. I also like that visually I can distinguish those 3 groups easily, instead of having to scan one by one a long list of endpoints. I'll keep this for this doc bugfix that will backport cleanly and we can return to this doc with another ticket, if that's ok.

Reply:
Absolutely! It was more of an observation and very much boils down to personal preference I believe :)